shared/js/ui/models/submit-breakage-form.es6.js

```diff
@@ -25,13 +25,15 @@ module.exports = function (category) {
     const blockedTrackers = collectAllUrls(aggregationStats.blockAction.list)
     const surrogates = collectAllUrls(aggregationStats.redirectAction.list)
     const urlParametersRemoved = this.tab.urlParametersRemoved ? 'true' : 'false'
+    const ctlYouTubeAdded = Boolean(this.tab.ctlYouTubeAdded)
```

Comment:
Nit: Is `Boolean` necessary here? I figure we know tab.ctlYouTubeAdded must be a bool right?

Reply:
Ah, good catch!